coderd/files/cache_test.go

```diff
@@ -100,21 +100,15 @@ func TestConcurrentFetch(t *testing.T) {
 	ctx := dbauthz.AsFileReader(testutil.Context(t, testutil.WaitShort))
 
 	// Expect 2 calls to Acquire before we continue the test
-	var (
-		hold sync.WaitGroup
-		wg   sync.WaitGroup
-	)
+	var wg sync.WaitGroup
 
+	wg.Add(2)
 	for range 2 {
-		hold.Add(1)
 		// TODO: wg.Go in Go 1.25
-		wg.Add(1)
 		go func() {
 			defer wg.Done()
-			hold.Done()
-			hold.Wait()
 			_, err := cache.Acquire(ctx, dbM, fileID)
-			require.NoError(t, err)
+			assert.NoError(t, err)
```

Comment:
looking at the docs for this, I suppose the change is so that the `require` variant doesn't try to terminate test execution from a goroutine, instead letting it run to completion?

just checking so I know for the future